test/fixtures/getDiff.fixture.txt

```diff
@@ -0,0 +1,55 @@
+diff --git a/package.json b/package.json
+index 4e5b84c..e0f922b 100644
+--- a/package.json
++++ b/package.json
+@@ -1,42 +1,29 @@
+ {
+   "name": "whats-my-line",
+   "version": "0.1.4",
```

Comment:
I think checking this in as the fixture would effectively mean we can't change version number or any deps in package.json without changing the expected test output.